src/Analyzers/CSharp/Tests/UseInterpolatedString/CSharpUseInterpolatedStringTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseInterpolatedString;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseInterpolatedString;
+
+using VerifyCS = CSharpCodeFixVerifier<
+    CSharpUseInterpolatedStringDiagnosticAnalyzer,
+    CSharpUseInterpolatedStringCodeFixProvider>;
+
+public sealed class CSharpUseInterpolatedStringTests
+{
+    [Fact]
+    public Task KeepInterpolatedStrings() => new VerifyCS.Test
+    {
+        TestCode = """
+        class Program
```

Comment:
Raw string should be idented 1 more level from property name in this and all remaining test cases. I guess you've copied it from somewhere and formatter didn't force-ident the raw string, it isn't that aggressive with their indentation